### PR TITLE
no_verify_blocking_bugs for EC, RC and first GA

### PR DESCRIPTION
**Pull request by ashwindasr**
`no_verify_blocking_bugs` for EC, RC and first GA

### CODE REVIEW

File: pyartcd/pyartcd/pipelines/promote.py [hunk 1]
```diff
@@ -259,6 +259,10 @@ async def run(self):
                 verify_flaws = True
                 if "prerelease" in impetus_advisories.keys() or assembly_type == assembly.AssemblyTypes.PREVIEW:
                     verify_flaws = False
+
+                if assembly_type in [assembly.AssemblyTypes.PREVIEW, assembly.AssemblyTypes.CANDIDATE] and self.assembly.endswith(".0"):
```
Comment (thegreyd) [inline pyartcd/pyartcd/pipelines/promote.py:263]: or self.assembly.endswith(".0") ?

Comment (ashwindasr) [reply pyartcd/pyartcd/pipelines/promote.py]: dang it, today's not my day.

fixed, repushed

**Comment by thegreyd** [comment]
Rebase should fix the failing unit test

**Pull request merged**